A3A/addons/scrt/Stringtable.xml

```diff
@@ -2957,636 +2957,742 @@
                 <Original>Airdrop Plane Destroyed</Original>
                 <Russian>Парашютно-десантный самолёт уничтожен</Russian>
                 <Chinesesimp>空投飞机被摧毁</Chinesesimp>
                 <Korean>공수 항공기가 파괴되었습니다</Korean>
+                <French>Destruction d'un avion de parachutage</French>
             </Key>
             <Key ID="STR_notifiers_airdrop_crew_killed">
                 <Original>Airdrop plane crew killed</Original>
                 <Russian>Экипаж парашютно-десантного самолёта уничтожен</Russian>
                 <Chinesesimp>空投机组人员阵亡</Chinesesimp>
                 <Korean>공수 항공기 조종사가 죽었습니다</Korean>
+                <French>L'équipage d'un avion de parachutage est mort</French>
             </Key>
             <Key ID="STR_notifiers_SUP_execute_marker">
                 <Original>%1. Target marked on map!</Original>
                 <Russian>%1. Цель отмечена на карте!</Russian>
                 <Chinesesimp>%1。目标在地图上被标记！</Chinesesimp>
                 <Korean>%1. 목표가 지도에 표시되었습니다!</Korean>
+                <French>%1. Cible marquée sur la carte !</French>
             </Key>
             <Key ID="STR_notifiers_SUP_execute_QRF">
                 <Original>A land %1 QRF just arrived</Original>
                 <Russian>Наземные силы быстрого реагирования %1 прибыли</Russian>
                 <Chinesesimp>%1 地面快速反应部队到达</Chinesesimp>
                 <Korean>%1의 지상 신속대응군이 방금 도착했습니다</Korean>
+                <French>QRF terrestre %1 vient d'arriver</French>
             </Key>
             <Key ID="STR_notifiers_SUP_execute_QRF_marker">
                 <Original>Land QRF</Original>
                 <Russian>Наземные СБР</Russian>
                 <Chinesesimp>地面快速反应部队</Chinesesimp>
                 <Korean>지상 신속대응군</Korean>
+                <French>QRF terrestre</French>
             </Key>
             <Key ID="STR_notifiers_SUP_execute_QRFAir">
                 <Original>A %1 air QRF just arrived</Original>
                 <Russian>Десантные силы быстрого реагирования %1 прибыли</Russian>
                 <Chinesesimp>%1 空中快速反应部队到达</Chinesesimp>
                 <Korean>%1의 공수부대 신속대응군이 방금 도착했습니다</Korean>
+                <French>Une QRF aérienne %1 vient d'arriver</French>
             </Key>
             <Key ID="STR_notifiers_SUP_execute_QRFAir_marker">
                 <Original>Air QRF</Original>
                 <Russian>Десантные СБР</Russian>
                 <Chinesesimp>空中快速反应部队</Chinesesimp>
                 <Korean>공수부대 신속대응군</Korean>
+                <French>QRF aérienne</French>
             </Key>
             <Key ID="STR_notifiers_SUP_execute_QRFAirdrop">
                 <Original>A %1 air mechanized QRF just arrived</Original>
                 <Russian>Десантные механизированные силы быстрого реагирования %1 прибыли</Russian>
                 <Chinesesimp>%1 空中机械化快速反应部队到达</Chinesesimp>
                 <Korean>%1의 공수 기계화 신속대응군이 방금 도착했습니다</Korean>
+                <French></French>
```

Comment:
Is this empty for a reason?

Reply:
Seems to be quite a few empty ones in that file

Reply:
It was to remind me to review the documents and correct some translations